google/datalab/bigquery/_view.py

```diff
@@ -186,7 +186,7 @@ def results(self, use_cache=True, dialect=None, billing_tier=None):
     """
     output_options = QueryOutput.table(use_cache=use_cache)
     return self._materialization.execute(output_options, dialect=dialect,
-                                               billing_tier=billing_tier).results
+                                               billing_tier=billing_tier).result()
```

Comment:
To match the base class, I made QueryJob.result a non-property.

How sure are you that it would call the base classes result()?  Not a huge python expert, but from my tests and what I could find the derived class' method overwrites the base class method.

result() doesn't have a defined return type, it is just whatever type the future returns, which is a table when the job is a query.